tests/test_workflow/test_methode_attribute.py

```diff
@@ -0,0 +1,148 @@
+from dessia_common import tests
+import unittest
+from dessia_common import core
+from dessia_common.workflow.blocks import GetModelAttribute, SetModelAttribute
+from dessia_common.workflow.core import Variable, TypedVariable
+import inspect
+from dessia_common.typings import AttributeType
+from dessia_common.forms import DessiaObject
+import time
+from typing import List, Dict
+
+class UnrealClassForTesting(DessiaObject):
+    def __init__(self, int_: int , variable_, float_: float = 0.0, str_: str = 'test', bool_: bool = True,
+                  l: List[int] = [1,2,3], d: Dict[str,int] = {'a':1,'b':2}, two: int = 2, name: str ='', **kwargs):
+        self.int_ = int_
+        variable_= variable_
+        self.float_ = float_
+        self.str_ = str_
+        self.bool_ = bool_
+        self.list_ = l
+        self.d = d
+        self.two = two
+        self.name = name
+        self.combine = self.int_*5
+        DessiaObject.__init__(self, **kwargs)
+    
+    def add_some_value(self) -> None:
+        self.combine = self.int_+self.two
+        return 
+
+
+class TestGetModelAttribute(unittest.TestCase):          
+    def setUp(self):
+        pass
+
+    def test_outputs(self):
+        instanciate = [(GetModelAttribute(AttributeType(UnrealClassForTesting,'int_')), 
```

Comment:
You should take advantage of of the @parameterized syntax.
Check in other test files to see how it is done

Reply:
I modified tests, and that's cleaner but that makes the number of test higher. Is it an issue?

Reply:
On the contrary, that is perfectly ok !